src/main/java/ch/njol/skript/structures/StructUsing.java

```diff
@@ -22,15 +21,15 @@
 })
 @Examples({
 	"using 1.21",
-	"using my-cool-addon-feature"
+	"using the experiment my-cool-addon-feature"
 })
 @Since("2.9.0")
 public class StructUsing extends Structure {
 
 	public static final Priority PRIORITY = new Priority(15);
 
 	static {
-		Skript.registerSimpleStructure(StructUsing.class, "using <.+>");
+		Skript.registerSimpleStructure(StructUsing.class, "(using|enable) [[the] experiment] <.+>");
```

Comment:
I think we should stick to using only, not enable
I don't want there to be any confusion about whether they do anythign differently, and it's easiest to just keep it uniform as `using`